lib/sonice/public/script.js

```diff
@@ -24,6 +29,71 @@ $.fn.keyboardShortcut = function() {
   })
 }
 
+/* So nice helpers */
+
+// Get a new artist image from Last.fm via jsonp
+// When found calls the `callback` with the image url as the first argument
+function artistImage(artist, callback) {
+  if (!artist) return callback();
+  cache = artistImage.cache;
+  artist = encodeURI(artist);
+  const getImageFromCache = 
+    function() { callback(cache[artist].random()); };
```

Comment:
After merging the `main` branch you can write it using the ES6 shorter syntax for functions:

```suggestion
  const getImageFromCache = () => { callback(cache[artist].random()); };
```